llvm/include/llvm/Analysis/LoopAccessAnalysis.h

```diff
@@ -33,6 +33,10 @@ class TargetTransformInfo;
 struct VectorizerParams {
   /// Maximum SIMD width.
   static const unsigned MaxVectorWidth;
+  /// Maximum unroll factor. Can represent actual unroll factor and/or some
+  /// other target-specific features, like LMUL factor for RISC-V with RVV
+  /// support.
+  static const unsigned MaxVectorUF;
```

Comment:
At the moment, we never interleave in EVL mode, why do we need to account for LMUL here?